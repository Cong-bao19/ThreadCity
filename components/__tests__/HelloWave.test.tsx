import React from 'react';
import renderer from 'react-test-renderer';
import { HelloWave } from '../HelloWave';

describe('HelloWave', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<HelloWave />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
