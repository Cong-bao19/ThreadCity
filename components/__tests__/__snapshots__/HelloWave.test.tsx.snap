// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWave renders correctly 1`] = `"ðŸ‘‹"`;
